lib/spoom/cli/commands/bump.rb

```diff
@@ -0,0 +1,85 @@
+# typed: false
+# frozen_string_literal: true
+
+require 'find'
+require 'open3'
+
+require_relative 'base'
+
+module Spoom
+  module Cli
+    module Commands
+      class Bump < Base
+        extend T::Sig
+
+        default_task :bump
+
+        desc "bump", "bump Sorbet sigils from `false` to `true` when no errors"
+        sig { params(directory: String, extension: String).returns(T::Array[String]) }
+        def bump(directory = ".", extension = "rb")
```

Comment:
The default values  for both the `directory` and `extension` should come from the content of `sorbet/config`.
You can use [this method](https://github.com/Shopify/spoom/blob/master/lib/spoom/sorbet.rb#L41) to get all of this done for you.

Reply:
Or change the `sorbet/config` or your test project to allow `.tmp` as extension. See how it's done here: https://github.com/Shopify/spoom/blob/master/test/spoom/cli/commands/config_test.rb#L124